bindings/py/cpp_src/bindings/math/py_Math.cpp

```diff
@@ -43,14 +42,6 @@ namespace nupic_ext
 {
 	void init_Math_Functions(py::module& m)
 	{
```

Comment:
happy to see those go.  There was one of them that had no std:: implementation in C++17